src/assets/styles/layout.scss

```diff
@@ -1,21 +1,15 @@
-//
-//  MEDIA QUERIES
-//––––––––––––––––––––––––––––––––––––––––––––––––––
+@import '~hds-design-tokens/lib/breakpoint/all.scss';
 
-// A map of breakpoints.
 $breakpoints: (
-  xs: 576px,
-  sm: 768px,
-  md: 992px,
-  lg: 1200px,
-  xlg: 1600px,
+  xs: $breakpoint-xs,
+  s: $breakpoint-s,
+  m: $breakpoint-m,
+  l: $breakpoint-l,
+  xl-minus: $container-width-xl,
+  xl: $breakpoint-xl,
+  xxl: 1600px, // $container-width-xl * 4 / 3
```

Comment:
if there is no support for this in HDS, do we need to support that width at all? 1600px is quite much.

Reply:
In events-monorepo there is 
```
$breakpoints: (
  xs: $breakpoint-xs,
  s: $breakpoint-s,
  m: $breakpoint-m,
  l: $breakpoint-l,
  xl: $breakpoint-xl,
  // xxl => 1568px
  xxl: $breakpoint-xl + $breakpoint-xs,
);
```